image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -273,26 +276,26 @@ spec:
         [< else ->]
         image: {{ ._rox.scanner.dbImage.fullRef | quote }}
         [< end ->]
+        env:
+          {{- include "srox.envVars" (list . "deployment" "scanner-db" "db") | nindent 10 }}
         ports:
-        - name: https-db
+        - name: postgresql
```

Comment:
Nitpick: All the other resources a nicely prefixed with `scanner-`, though this could also be called `scanner-db-port`? Do we reference this name in the service definition?

Reply:
actually, looking at the ports in Scanner, they are called `https` and `grpc` so we seem to name them in the deployment definitions based on the protocol, so i'm going to keep this as `postgres`

Reply:
I think we want to do tcp-postgresql to make Istio happy if it is used

Reply:
good point about istio i didnt know that. i'll switch it to `tcp-postgres` just to be safe. Looks like Istio https://istio.io/latest/docs/reference/config/analysis/ist0118/ wants `<protocol>[-<suffix>]`